presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcTableLayoutHandle.java

```diff
@@ -14,46 +14,42 @@
 package com.facebook.presto.plugin.jdbc;
 
 import com.facebook.presto.spi.ColumnHandle;
-import com.facebook.presto.spi.ConnectorPartition;
+import com.facebook.presto.spi.ConnectorTableLayoutHandle;
 import com.facebook.presto.spi.predicate.TupleDomain;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
 
-import static com.google.common.base.MoreObjects.toStringHelper;
 import static java.util.Objects.requireNonNull;
 
-public class JdbcPartition
-        implements ConnectorPartition
+public class JdbcTableLayoutHandle
+    implements ConnectorTableLayoutHandle
```

Comment:
The original indent of 8 spaces for `implements` was correct. See code style here with formatting rules for IntelliJ: https://github.com/airlift/codestyle